.github/workflows/review-check.yml

```diff
@@ -0,0 +1,56 @@
+name: Enforce Review Rules
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  enforce-review-rules:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Get list of files changed
+      id: files
+      run: |
+        echo "::set-output name=files::$(git diff --name-only ${{ github.event.before }} ${{ github.sha }} | tr '\n' ' ')"
+    
+    - name: Check if PR modifies specific path
+      id: check-path
+      run: |
+        if echo "${{ steps.files.outputs.files }}" | grep -q '^src/'; then
+          echo "::set-output name=specific_path::true"
+        else
+          echo "::set-output name=specific_path::false"
+        fi
```

Comment:
Wouldn't it be better to merge these two steps?
Then there would be no need for setting the intermediate `files` outputs.
Also, this checks that the diffed files start with `src/`, but wouldn't it fail if the output was `docs/foo.md src/bar.sol`?

Reply:
Merged and fixed